roles/postgresql/defaults/main.yml

```diff
@@ -6,7 +6,8 @@ postgresql_socket_path: "{{ postgresql_base_path }}/sockets"
 postgresql_config_path: "{{ postgresql_base_path }}/config"
 postgresql_connect_socket: true
 
-postgresql_container_version: "13.11"
+postgresql_container_version: "13"
+postgresql_container_custom_version: ~
```

Comment:
What is this override needed for? People can just set `postgresql_container_version`, and it'll override the default from the role here